src/main/java/racingcar/view/ResultView.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.view;
+
+import racingcar.RacingResult;
+import util.StringUtils;
+
+import java.util.List;
+
+/**
+ * @author daheekim
+ * @version : 0.0.0
+ */
+public class ResultView {
+    private static final String CR_LF = "\n";
+    private static final String RACE_TRACK = "-";
+
+    public static void print(RacingResult racingResult) {
+        System.out.println(CR_LF + "실행 결과");
+
+        for (int attempt = 0; attempt < racingResult.getAttempt(); attempt++) {
+            printResultByAttempt(racingResult.getResultByAttempt(attempt));
+        }
```

Comment:
for문을 직접 정의하기보단 가급적 stream을 사용해주세요! 혹은 이렇게 사용하더라도

```suggestion
        for (int attempt = 0, size = racingResult.getAttempt(); attempt < size; attempt++) {
            printResultByAttempt(racingResult.getResultByAttempt(attempt));
        }
```
이런식으로 길이를 미리 정의해놓고 사용하는게 좋습니다.

Reply:
사실 배열이나 컬렉션 없이 횟수정보(racingResult.getAttempt())를 이용해서 
어떻게 스트림을 돌릴 수 있을까 생각해보지 못했는데 리뷰어님의 코멘트 덕분에 고민해보고 시도해봤습니다.

Stream.iterate 메소드로 for문과 같은 동작을 실행시킬 수 있네요 :)